src/wp-admin/includes/nav-menu.php

```diff
@@ -750,31 +852,35 @@ function wp_nav_menu_item_taxonomy_meta_box( $object, $box ) {
 				<?php submit_button( __( 'Search' ), 'small quick-search-submit hide-if-js', 'submit', false, array( 'id' => 'submit-quick-search-taxonomy-' . $taxonomy_name ) ); ?>
 			</p>
 
-			<ul id="<?php echo $taxonomy_name; ?>-search-checklist" data-wp-lists="list:<?php echo $taxonomy_name?>" class="categorychecklist form-no-clear">
+			<ul id="<?php echo $taxonomy_name; ?>-search-checklist" data-wp-lists="list:<?php echo $taxonomy_name; ?>" class="categorychecklist form-no-clear">
 			<?php if ( ! empty( $search_results ) && ! is_wp_error( $search_results ) ) : ?>
 				<?php
 				$args['walker'] = $walker;
-				echo walk_nav_menu_tree( array_map('wp_setup_nav_menu_item', $search_results), 0, (object) $args );
+				echo walk_nav_menu_tree( array_map( 'wp_setup_nav_menu_item', $search_results ), 0, (object) $args );
 				?>
 			<?php elseif ( is_wp_error( $search_results ) ) : ?>
 				<li><?php echo $search_results->get_error_message(); ?></li>
 			<?php elseif ( ! empty( $searched ) ) : ?>
-				<li><?php _e('No results found.'); ?></li>
+				<li><?php _e( 'No results found.' ); ?></li>
 			<?php endif; ?>
 			</ul>
 		</div><!-- /.tabs-panel -->
 
 		<p class="button-controls wp-clearfix">
 			<span class="list-controls">
-				<a href="<?php
-					echo esc_url(add_query_arg(
-						array(
-							$taxonomy_name . '-tab' => 'all',
-							'selectall' => 1,
-						),
-						remove_query_arg($removed_args)
-					));
-				?>#taxonomy-<?php echo $taxonomy_name; ?>" class="select-all aria-button-if-js"><?php _e( 'Select All' ); ?></a>
+				<a href="
+				<?php
```

Comment:
newline in `href`

Reply:
Agreed. Move `<?php` backwards to old line.

Reply:
Fixed with backport of 46240